Jenkinsfile

```diff
@@ -29,6 +30,73 @@ pipeline {
                 sh 'build/run make -j\$(nproc) test-integration'
             }
         }
+
+        stage('Integration Tests SetUp') {
+            steps {
+                sh 'tests/scripts/saveImages.sh'
+                stash name: 'repo', useDefaultExcludes: false
+            }
+        }
+
+        stage('Integration Tests - parallel runs') {
+            environment {
+                KUBE_VERSION = "v1.6"
+            }
+            steps {
+                parallel(
+                    "GCE_K8s_v1.7" : {
+                        node("gce_vm"){
+                            script {
+                                try{
+                                    withEnv(["KUBE_VERSION=v1.7"]){
+                                        unstash 'repo'
+                                        deleteDir()
+                                        sh "docker load -i rookamd64.tar"
+                                        sh "docker load -i toolboxamd64.tar"
+                                        sh 'build/run make -j\$(nproc) build.common'
+                                        sh 'tests/scripts/kubeadm-dind.sh up'
+                                        sh 'build/run make -j\$(nproc) test-integration'
+                                    }
+                                }
+                                finally{
```

Comment:
given the we are using kernel rbd, and kernel cephfs its conceivable that we are going to leave state behind on the slave/agent that interferes with future tests. can we just each slave for "integration tests" once? i.e. restart it at the VM/cloud level? that would remove the need to cleanup here.